src/components/request/RequestHero.tsx

```diff
@@ -31,14 +32,25 @@ const RequestHero: FC<Props> = ({ chainId }) => {
     chainName = CHAINS[chainId].name;
   }
   const { oracle, state } = useClient();
-  const { requestState, parsedIdentifier } = useReader(state);
+  const { requestState, parsedIdentifier, requestTx, explorerUrl } =
+    useReader(state);
 
   return (
     <HeroSection>
       <HeroContentWrapper>
         <HeroHeaderRow>
           <HeaderTitle>
             {parsedIdentifier || "Optimistic Oracle Request"}
+            <RequestTxText>
+              Request:{" "}
```

Comment:
requestTx may not exist, or it may potentially take a while to find it. perhaps we should hide this until it is found?

Reply:
Seems fine to me!